include/deal.II/lac/affine_constraints.h

```diff
@@ -1658,15 +1658,24 @@ class AffineConstraints : public Subscriptor
     /**
      * Copy constructor.
      */
-    template <typename ConstraintLineType>
-    ConstraintLine(const ConstraintLineType &other);
+    ConstraintLine(const ConstraintLine &other) = default;
+
+    /**
+     * Move constructor.
+     */
+    ConstraintLine(ConstraintLine &&other) = default;
```

Comment:
We should mark this as `noexcept` since it cannot throw an exception.

Reply:
So done!